apis/fluentbit/v1alpha2/fluentbitconfig_types.go

```diff
@@ -23,8 +23,8 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// NamespaceFluentBitSpec defines the desired state of FluentBit
-type NamespaceFluentBitSpec struct {
+// NamespacedFluentBitSpec defines the desired state of FluentBit
+type NamespacedFluentBitSpec struct {
```

Comment:
One minor change: NamespacedFluentBitSpec => NamespacedFluentBitCfgSpec because it's for FluentBitConfig CRD :)

Reply:
Ah, my bad. Fixed

Reply:
Lemme know if there's anything else I might have missed. If this looks good feel free to merge